src/odemis/gui/comp/stream_panel.py

```diff
@@ -419,7 +419,12 @@ def _on_colormap_value(self, colour):
         else:  # Can't find the colour => it's custom tint
             if isinstance(colour, tuple):
                 self.colormap_choices[TINT_CUSTOM_TEXT] = colour
-                self.combo_colormap.SetClientData(self._colormap_customtint_idx, colour)
+                # sometimes switching to a custom color will throw a C/C++ wrapper runtime error
+                # this can be ignored and won't affect the colormap selected
+                try:
+                    self.combo_colormap.SetClientData(self._colormap_customtint_idx, colour)
+                except RuntimeError:
```

Comment:
I discussed this error with @pieleric and checked the log https://support.delmic.com/scp/tickets.php?id=1453#reply 
From this log, there are dead objects around this error, it seems there is something deleted in the gui, like a stream panel etc but not taken into account correctly, leading to this runtime error.

Reply:
Indeed, but there is not stream panel deleted, only the color picker box is deleted when clicking ok, I guess this does not get handle properly. Indeed I have also seen the dead objects. Do you think we can address this issue/error in a better way?

Reply:
Now we do not understand what behavior in the GUI(that is the clicks) throw this error. If we can point pot when this error is thrown out, we could catch it better?

Reply:
I can change the pass line into a logging.warning to log it either way and we could spend more time on it later on? I can in this case create a Jira task on the SWM board for it. @tepals do you think this is a valid temporary solution?

Reply:
I think it'd be better to find out how to reproduce this issue, and then properly fix it by unsubscribing to the tint.

Reply:
Please note that for now it's not pretty but it *only* causes an error message in the log. I'm pretty certain it doesn't cause any actual issue in the behaviour of the GUI, so not super urgent.